packages/plugin-payment-ui/src/components/form/ConfigForm.tsx

```
@@ -0,0 +1,163 @@
+                  <p>
+                    {__(
+                      'Register following URL in your payment provider or bank'
+                    )}
+                  </p>
+                }
+                <FormControl defaultValue={callbackUrl} disabled={true} />
+              </FormGroup>
+            </>
+          )}
+
+          {metaData && metaData.link && (
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)